### PR TITLE
document breaking change between cy.route & cy.intercept

### PR DESCRIPTION
**Pull request by samijaber**
Closes https://github.com/cypress-io/cypress-documentation/issues/3403. See https://github.com/cypress-io/cypress/issues/9302#issuecomment-745518565

### CODE REVIEW

File: source/guides/references/migration-guide.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ cy.wait('@createUser')
   })
 ```
 
+## overriding route matchers
+Unlike `cy.route`, `cy.intercept` currently does _not_ allow you to override a previous interceptor. For more information on this, see https://github.com/cypress-io/cypress/issues/9302 and https://glebbahmutov.com/blog/cypress-intercept-problems/#no-overwriting-interceptors.
+
```
Comment (jennifer-shehane) [inline source/guides/references/migration-guide.md:54]: The placement of this is in between a before/after example. This should be it's own section, not in the middle of another section.

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]

File: source/guides/references/migration-guide.md (hunk 1 above)
Comment (samijaber) [reply source/guides/references/migration-guide.md]: Oops, can't believe I missed that. Fixed 👍

File: source/guides/references/migration-guide.md [hunk 2]
```diff
@@ -83,6 +83,9 @@ cy.intercept('GET', '/projects', {
 })
 ```
 
+## overriding route matchers
+Unlike `cy.route`, `cy.intercept` currently does _not_ allow you to override a previous interceptor. For more information on this, see https://github.com/cypress-io/cypress/issues/9302 and https://glebbahmutov.com/blog/cypress-intercept-problems/#no-overwriting-interceptors.
```
Comment (flotwig) [inline source/guides/references/migration-guide.md:87]:
```suggestion
Unlike `cy.route`, `cy.intercept` currently does _not_ allow you to override a previous response. For more information on this, see https://github.com/cypress-io/cypress/issues/9302 and https://glebbahmutov.com/blog/cypress-intercept-problems/#no-overwriting-interceptors. Overriding responses will be added in a future release.
```

File: source/api/commands/intercept.md [hunk 3]
```diff
@@ -22,6 +22,7 @@ Unlike {% url "`cy.route()`" route %}, `cy.intercept()`:
 * does not require calling {% url "`cy.server()`" server %} before use - in fact, `cy.server()` does not influence `cy.intercept()` at all.
 * does not have method set to `GET` by default, but intercepts `*` methods.
 * uses plain substring match, or RegExp, or {% url minimatch %} to match URL.
+* cannot override previous interceptors: see https://github.com/cypress-io/cypress/issues/9302 and https://glebbahmutov.com/blog/cypress-intercept-problems/#no-overwriting-interceptors for more information.
```
Comment (flotwig) [inline source/api/commands/intercept.md:25]:
```suggestion
* Currently, cannot override previously-defined responses: see https://github.com/cypress-io/cypress/issues/9302 and https://glebbahmutov.com/blog/cypress-intercept-problems/#no-overwriting-interceptors for more information. Overriding responses will be added in a future release.
```

**Review by flotwig — APPROVED** [review]
LGTM, updated to note that support is planned

**Review by flotwig — APPROVED** [review]

**Pull request merged**